plans/dht/README.md

```diff
@@ -0,0 +1,70 @@
+# `Plan:` The go-libp2p DHT behaves
+
+IPFS can safely rely on the latest DHT upgrades by running go-libp2p DHT tests directly
+
+## What is being optimized (min/max, reach)
+
+- (Minimize) Numbers of peers dialed to as part of the call
+- (Minimize) Number of failed dials to peers
+- (Minimize) Time that it took for the test
+- (Minimize) Routing efficiency (# of hops (XOR metric should decrease at every step))
+
+## Plan Parameters
+
+- **Network Parameters**
+  - `Region` - Region or Regions where the test should be run at (default to single region)

```

Comment:
We can remove the region as a parameter here, since we won't be doing cross-region anything.